docs/implementation.md

```diff
@@ -402,7 +402,7 @@ POST /transactions
 #### Request body schema
 ```json
 {
-  "fee": "A number representing the transaction fee to be paid to write this transaction to the blockchain.",
+  "minFee": "A number representing the minimum transaction fee to be paid to write this transaction to the blockchain. The actual fee paid can fluctuate deterministically.",
```

Comment:
Just a reminder that this requires a corresponding change in ion repo as well.

Reply:
Good call. I created an issue so we remember what we've done when we update ion

https://github.com/decentralized-identity/ion/issues/56